.github/workflows/workflow.yaml

```diff
@@ -374,10 +374,11 @@ jobs:
             echo "::set-output name=terraformdocsconfig::.terraform-docs.yaml"
           else
             echo "Using workflow configuration for commitlint"
-            echo "::set-output name=terraformdocsconfig::.github-workflow-terraform-config/.terraform-docs.yaml"
+            echo "::set-output name=terraformdocsconfig::/home/runner/work/.github-workflow-terraform-config/.terraform-docs.yaml"
           fi
       - name: Render terraform docs and push
         uses: terraform-docs/gh-actions@v1.0.0
+        if: ${{ github.event_name == 'push' }}
```

Comment:
Values in `if` are implicitly expressions, so we don't actually need the enclosing brackets `${{ ... }}`

Reply:
oh? damn.. hmm feil i docen da :(
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-contexts

Reply:
jeg fikk faktisk en fail på å ikke bruke ${{ }} tidligere. derfor jeg la det på igjen. litt rart